docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -118,14 +118,14 @@ virt-edit -a rhcos-${RHCOS_VERSION}.qcow2 \
 
 exo storage upload rhcos-${RHCOS_VERSION}.qcow2 "sos://${CLUSTER_ID}-bootstrap" --acl public-read
 
-exo vm template register "rhcos-${RHCOS_VERSION}" \
+exo compute instance-template register "rhcos-${RHCOS_VERSION}" \
```

Comment:
Do we the minimum `exo` version which supports this new command? It's definitely not 1.28.0 (as listed in the prerequisites):

```simon@phoenix:~ $ exo version
exo 1.28.0 bf8f7aac (egoscale 0.47.0)
simon@phoenix:~ $ exo compute instance-template
error: unknown command "compute" for "exo"
```

Reply:
Ok found it 👍

[`1.37.0`](https://github.com/exoscale/cli/blob/v1.49.3/CHANGELOG.md#1370)